Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/FarePolicy.hs

```diff
@@ -45,6 +45,9 @@ data FarePolicyD (s :: DTC.UsageSafety) = FarePolicy
     tollCharges :: Maybe HighPrecMoney,
     perMinuteRideExtraTimeCharge :: Maybe HighPrecMoney,
     congestionChargeMultiplier :: Maybe CongestionChargeMultiplier,
+    perDistanceUnitInsuranceCharge :: Maybe HighPrecMoney,
+    cardChargeMultiplier :: Maybe Double,
```

Comment:
Instead of having 2 columns for cardCharge, it should be one with type `FixedCardChage HighPrecMoney | ProgressiveCardCharge Double` , just like we have for night charge, waiting charge etc.

Reply:
Fixed, created a object type, which has both fields as maybe, i.e. `perDistanceUnitMultiplier` and `fixed`.